test/WebJobs.Extensions.CosmosDB.Tests/Trigger/CosmosDBTriggerAttributeBindingProviderTests.cs

```diff
@@ -359,6 +359,8 @@ public async Task ValidChangeFeedOptions_Succeed(ParameterInfo parameter)
             Assert.Equal(new Uri("https://fromSettings"), binding.LeaseCollectionLocation.Uri);
             Assert.Equal(10, binding.ChangeFeedProcessorOptions.MaxItemCount);
             Assert.True(binding.ChangeFeedProcessorOptions.StartFromBeginning);
+            Assert.NotNull(binding.ChangeFeedProcessorOptions.StartTime);
+            Assert.Equal(binding.ChangeFeedProcessorOptions.StartTime, DateTime.Parse("2020-11-25T22:36:29Z"));
```

Comment:
This needs a separate test, cannot be the same as setting StartFromBeginning

Reply:
Added few combinations